frontend/src/pages/exploreApplication/EnableModal.tsx

```diff
@@ -53,6 +60,7 @@ const EnableModal: React.FC<EnableModalProps> = ({ selectedApp, shown, onClose }
       selectedApp.spec.shownOnEnabledPage = true;
       /* eslint-enable no-param-reassign */
 
+      setIsEnableValuesHasEmptyValue(true);
       onClose();
```

Comment:
Is this line necessary? It looks like once the app is enabled, it's going to close itself. 
Also I don't believe the value is being cleared here either, so it looks a little confusing to see code saying it's being set to empty

Reply:
The `isEnableValueHasEmptyValue` is the variable to control the disability of the 'Submit' button, it need to be reset once the modal is closed.

Reply:
Gotcha, I see, the modal doesn't have conditional rendering and instead is returning undefined when it is hidden..